egs2/voxceleb/spk1/conf/tuning/train_RawNet3_sgdr_bs512_da_sampler_adam.yaml

```diff
@@ -0,0 +1,73 @@
+# RawNet3 reproduce recipe configuration.
+
+frontend: raw
+
+encoder: rawnet3
+encoder_conf:
+  model_scale: 8
+  ndim: 1024
+  sinc_stride: 16
+
+pooling: chn_attn_stat
+pooling_conf:
+  input_size: 1536  # 1.5 * ndim of RawNet3 encoder
+
+projector: rawnet3
+projector_conf:
+  input_size: 3072  # 2 * input_size of pooling
+  output_size: 256
+
+preprocessor: spk
+preprocessor_conf:
+  target_duration: 3.0  # seconds
+  sample_rate: 16000
+  num_eval: 5
+  noise_apply_prob: 0.5
+  noise_info:
+  - [1.0, '/home/jeeweonj/corpora/voxcelebs/musan_speech.scp', [4, 7], [13, 20]] 
+  - [1.0, '/home/jeeweonj/corpora/voxcelebs/musan_noise.scp', [1, 1], [0, 15]] 
+  - [1.0, '/home/jeeweonj/corpora/voxcelebs/musan_music.scp', [1, 1], [5, 15]] 
```

Comment:
Can you replace this with relative paths?

Reply:
It might be better to prepare these scp files in `local/data.sh` and point to them here.